app/models/task.rb

```diff
@@ -1,4 +1,6 @@
 class Task < ApplicationRecord
+  validates_with TaskValidator
   validates :todo, presence: true
+  validates_length_of :todo, :maximum => 300
```

Comment:
```suggestion
  validates_length_of :todo, maximum: 300
```

I think we can do something like:
`validates_length_of :todo, maximum: 300, allow_blank: false`

And we can rid of `validates :todo, presence: true`

But you should check it